src/main/scala/mesosphere/marathon/event/http/HttpEventActor.scala

```diff
@@ -96,7 +96,13 @@ class HttpEventActor(conf: HttpEventConfiguration,
     //remove all unsubscribed callback listener
     limiter = limiter.filterKeys(subscribers.urls).iterator.toMap.withDefaultValue(NoLimit)
     metrics.skippedCallbacks.mark(limited.size)
-    active.foreach(post(_, event, self))
+    active.foreach(url => Try(post(url, event, self)) match {
```

Comment:
The post method itself will send a NotificationFailed in this case. 
Do I miss something?